antarest/matrixstore/model.py

```diff
@@ -0,0 +1,63 @@
+import enum
+from typing import List, Any, Optional
+
+from dataclasses import dataclass
+from dataclasses_json import DataClassJsonMixin  # type: ignore
+from sqlalchemy import Column, String, Enum, DateTime  # type: ignore
+
+from antarest.common.persistence import Base
+
+
+class MatrixType(enum.IntEnum):
+    INPUT = 1
+    OUTPUT = 2
+
+
+class MatrixFreq(enum.IntEnum):
+    HOURLY = 1
+    DAILY = 2
+    WEEKLY = 3
+    MONTHLY = 4
+    ANNUAL = 5
+
+
+class Matrix(Base):  # type: ignore
```

Comment:
a name field would be nice so the user can find easily his matrix.
also we could add a metadata table so that user can add tags to matrix and query them to retrieve a list of matrix

Reply:
for the metadata part, this will be in an other PR (issue : #370 )

Reply:
Matrix will be shared accros study, a name will be ambigus. Moreover matrix store is like a matrix bucket for storage module. It should be used by storage for its purpose but not by user as standalone

Reply:
no, actually it will be used by user. they will create matrix with name (like 'wind fc scenarioA', 'wind fc scenario B' and so on) so that they can find them easily later on and assign them to study inputs.

Reply:
actually (this wasn't specified in the specs) this lead me to think that matrix should have an owner. we could then add a unique (name x owner) unique constraint. (and group assignement but i'll create another issue for group later, since this will imply other management endpoints)